erpnext/regional/doctype/gstr_3b_report/gstr_3b_report.html

```diff
@@ -0,0 +1,245 @@
+<style>
+	.print-format {
+		padding: 15mm;
+		font-size: 8.0pt !important;
+		font-family: Tahoma, sans-serif;
+	}
+	.disabled {
+		background-color: #d9d9d9;
+	}
+
+</style>
+<h3 class="text-center">GSTR3B-Form</h3>
+<h5>GSTIN:&nbsp{{ data.gstin }}</h5>
+
+<h5>3.1&nbsp&nbspDetails of Outward Supplies and inward supplies liable to reverse charge</h5>
```

Comment:
Better to use `&nbsp;`

And have a space after it too, helps with readability.